src/server/services/MediaResourceServices.js

```diff
@@ -18,7 +18,7 @@ class MediaResourceServices {
     );
 
     const musicresCount = await MediaQuery.getMusicResCountByCourse(
-      courseLabel
+      '201-MSCIND55-1'
```

Comment:
@maxpluez For the final code that goes into master, this should not be hardcoded.